salt/metalk8s/kubeadm/init/kubeconfig/admin.sls

```diff
@@ -0,0 +1,12 @@
+Create kubeconf file for admin:
+  metalk8s_kubeconfig.managed:
+    - name: /etc/kubernetes/admin.conf
+    - client_cert_info:
+        signing_private_key: /etc/kubernetes/pki/ca.key
+        signing_cert: /etc/kubernetes/pki/ca.crt
+        CN: kubernetes-admin
+        O: system:masters
+    - config_context:
+        master_address: https://10.200.5.53:6443
```

Comment:
Seems really hardcoded here.

And it's not master_address it the kube-apiserver address I think.

Reply:
oups, will change it